Handling Zarr and Dask parallell processing

### What is your issue?

FAILED tests/standardise/test_standardise.py::test_standardise_footprint_different_chunking_schemes - ValueError: Specified zarr chunks encoding['chunks']=(2,) for variable named 'air_pressure' would overlap multiple dask chunks ((2, 1),) on the region (slice(3, None, None),). Writing this array in parallel with dask could lead to corrupted data. Consider either rechunking using `chunk()`, deleting or modifying `encoding['chunks']`, or specify `safe_chunks=False`.


FAILED tests/store/storage/test_localzarrstore.py::test_match_chunking - ValueError: Specified zarr chunks encoding['chunks']=(16,) for variable named 'mf' would overlap multiple dask chunks ((16, 16, 16, 16, 16, 10),) on the region (slice(31, None, None),). Writing this array in parallel with dask could lead to corrupted data. Consider either rechunking using `chunk()`, deleting or modifying `encoding['chunks']`, or specify `safe_chunks=False`.


FAILED tests/store/storage/test_localzarrstore.py::test_for_missing_data_from_appending_in_loop - ValueError: Specified zarr chunks encoding['chunks']=(403,) for variable named 'y' would overlap multiple dask chunks ((403, 403, 14),) on the region (slice(820, None, None),). Writing this array in parallel with dask could lead to corrupted data. Consider either rechunking using `chunk()`, deleting or modifying `encoding['chunks']`, or specify `safe_chunks=False`.

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -7,6 +7,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased](https://github.com/openghg/openghg/compare/0.10.1...HEAD)
 
+### Fixed
+
+- Fixed options used with `xr.Dataset.to_zarr` in reponse to updates in xarray. [PR #1160](https://github.com/openghg/openghg/pull/1160)
+
 ### Added
 
 - Removed parsers that are unused. [PR #1129](https://github.com/openghg/openghg/pull/1129)
@@ -15,10 +19,6 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Updated
 
 - Updated ICOS standardise function to reflect changes in ASCII file format. [PR #1140](https://github.com/openghg/openghg/pull/1140)
-
-
-### Updated
-
 - Added `rename_vars` option to `get_obs_surface` to allow variable names based around species to be returned. [PR #1130](https://github.com/openghg/openghg/pull/1130)
 - Added option to `get_obs_column` to return the column data directly rather than converting to mole fractions. [PR #1131](https://github.com/openghg/openghg/pull/1131)
 

--- a/openghg/store/storage/_localzarrstore.py
+++ b/openghg/store/storage/_localzarrstore.py
@@ -166,6 +166,7 @@ class LocalZarrStore(Store):
                 append_dim=append_dim,
                 compute=True,
                 synchronizer=zarr.ThreadSynchronizer(),
+                safe_chunks=False,
             )
         # Otherwise we create a new zarr Store for the version
         else:

